atom/common/platform_util_linux.cc

```diff
@@ -13,7 +13,8 @@
 
 namespace {
 
-void XDGUtil(const std::string& util, const std::string& arg) {
+void XDGUtil(const std::string& util, const std::string& arg,
```

Comment:
We don't actually need the exit code, we only need to know whether the operation succeeded. So `XDGUtil` can just return `bool` to indicate whether the operation succeeded.